src/routes/write-exercise/components/__tests__/Actions.spec.tsx

```diff
@@ -29,9 +33,8 @@ describe('Components', () => {
         isFinished: true
       }
 
-      const component = shallow(<Action {...actionProps} />)
-      expect(component.find('Button')).toHaveLength(1)
-      expect(component.find('[testID="check-out"]')).toHaveLength(1)
+      const { getByText } = render(<Action {...actionProps} />, { wrapper: wrapWithTheme })
+      expect(getByText(labels.exercises.showResults)).toBeDefined()
```

Comment:
:+1: Seems like you really have the hang of testing now and you even like it kind of? :D Good job!